connector/launchers/runtime-embedded/build.gradle.kts

```diff
@@ -25,6 +39,14 @@ dependencies {
     runtimeOnly(libs.edc.bom.controlplane.sql)
     runtimeOnly(libs.edc.bom.dataplane.sql)
     implementation(libs.edc.iam.mock)
+
+    runtimeOnly(libs.edc.fc.core)
+    runtimeOnly(libs.edc.fc.ext.api)
+    implementation(libs.edc.fc.spi.crawler)
```

Comment:
nit: this dependency can be likely removed (spi are needed for implementations, but there aren't any in this launcher)